src/lib/taggingUtils.ts

```diff
@@ -63,15 +63,23 @@ const taggingUtils = {
     return Object.keys(tagObject).map(tagKey => this.buildTag(tagKey, tagObject[tagKey]));
   },
 
-  generateCustomTags(annotationList: string[] = []): string[] {
-    return annotationList.map(el => this.buildTag(ANNOTATION_LABEL, el));
+  generateCustomTags({
+    annotations = [],
+    useFallback = false,
+  }: {
+    annotations: string[];
+    useFallback?: boolean;
+  }): string[] {
+    return useFallback
+      ? annotations.map(el => this.buildTag(ANNOTATION_LABEL, el, true))
```

Comment:
Why don't you just return `annotations.map(el => this.buildTag(ANNOTATION_LABEL, el, useFallback))`?

Reply:
Because I didn't think of it ;-)

Reply:
Done with https://github.com/d4l-data4life/js-sdk/pull/45/commits/9988611866cac92a7f6972b9841917950a019f07